cmd/image-builder/main.go

```diff
@@ -125,12 +125,7 @@ func main() {
 				"method":     values.Method,
 				"status":     values.Status,
 				"latency_ms": values.Latency.Milliseconds(),
-			}
-			rid := c.Request().Header.Get("X-Rh-Edge-Request-Id")
-			if rid != "" {
-				fields["request_id"] = strings.Replace(rid, "\n", "", -1)
-			} else {
-				fields["request_id"] = random.String(12)
+				"request_id": c.Request().Context().Value(requestIdCtx),
 			}
```

Comment:
This was moved into the middleware, use the value instead of extracting it again.

Reply:
How come these do not work? :thinking: